site/src/Librecores/ProjectRepoBundle/Entity/User.php

```diff
@@ -94,6 +94,14 @@ class User extends BaseUser
      **/
     protected $organizationMemberships;
 
+    /**
+     * @var ArrayCollection
+     *
+     * @ORM\OneToMany(targetEntity="ProjectClassification", mappedBy="parentUser")
+     * @ORM\JoinColumn(name="projectId", referencedColumnName="id")
```

Comment:
No need to explicitly define join column unless it is ambiguous, let Symfony manage however it seems fit.